deploy_local_testnet.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -i
+
+
+echo "0. Changing working directory as home folder"
+cd ~
```

Comment:
Just `cd`.

Reply:
if the scirpt does not let choice for the location, I suggest not using home, but something like `~/.gno/testnet`